.github/workflows/build-dev-branches.yml

```diff
@@ -0,0 +1,37 @@
+name: Build Dev Branches
+
+on: [push, workflow_dispatch]
```

Comment:
Shall we add `branches-ignore` entry here to ignore the `main` branch pushes?

Not mandatory, but we already have a build workflow for the `main` branch, right?

Reply:
Fixed in 7fc58c96286a6ccf8f967f94e633efd885bcade0